javascript/index.js

```diff
@@ -32,14 +52,26 @@ function printMilliseconds() {
 
 function printSplit() {
   // ... your code goes here
+  var childElement = document.createElement("LI");
+  childElement.innerHTML = chronometer.splitClick()
+  splits.appendChild(childElement);
+
 }
 
 function clearSplits() {
   // ... your code goes here
+ 
```

Comment:
You don't necessarily have to check if there are nodes. Also another way you can reset the splits is by using `splits.innerHTML = ' '`